code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/device/autoimplanter
+	name = "autoimplanter"
+	desc = "A device that automatically injects a cyber-implant into the user without the hassle of extensive surgery. It has a slot to insert implants and a screwdriver slot for removing accidentally added implants."
+	icon_state = "gangtool-white"
+	item_state = "walkietalkie"
+	w_class = 2
+	var/obj/item/organ/internal/cyberimp/storedorgan
+
+/obj/item/device/autoimplanter/attack_self(mob/user, obj/item/I)//when the object it used...
+	if(!storedorgan)
+		user << "<span class='notice'>[src] currently has no implant stored.</span>"
+		return
+	storedorgan.Insert(user)//insert stored organ into the user
+	user << "<span class='notice'>You feel a slight tickle as [src] quickly implants you with the [storedorgan].</span>"
+	storedorgan = null
+
+/obj/item/device/autoimplanter/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/organ/internal/cyberimp))
+		if(storedorgan)
+			user << "<span class='notice'>[src] already has an implant stored.</span>"
+			return
+		user.unEquip(I)
```

Comment:
do i replace this line with that or add that under this line? also, what exactly does it do?